packages/api-server/src/methods/validator.ts

```diff
@@ -20,6 +20,8 @@ export function middleware(
     params: any[] = [],
     cb: (err: any, val?: any) => void
   ) {
+    console.log("[validator middleware] -", params);
```

Comment:
We should remove this in the production environment. The content of a transaction may be very large.

Reply:
https://github.com/nervosnetwork/godwoken-web3/pull/4/commits/4c6f5e3746b39bd4d8f5181c3ed906164aeb7b45